### DISCUSSION TITLE
How can I indent the root folder label and also indent its children?

### DISCUSSION THREAD

**Discussion by charbelnicolas** [nvim-tree/nvim-tree.lua#2022] [Q&A]
Hello, I've been a neo-tree user for the longest time and I'm switching to NvimTree. I want to know if it is possible to indent the root folder 1 space and also indent its children 1 space?

This is how it looks like:

![20230224_163123](https://user-images.githubusercontent.com/14960414/221297149-883875ee-8c6f-447f-8106-4b70c70c5b0f.png)

And this is how I want it (this is Neo Tree):

![20230224_163432](https://user-images.githubusercontent.com/14960414/221297582-ba56c46b-31ab-4457-8552-6ba8c2b84e0e.png)

It looks a bit nicer IMO

**Comment by charbelnicolas** [comment]
I don't want to use the signcolumn or number column, just a single space " ".

**Comment by alex-courtis** [comment]
> indent the root folder 1 space

I'm not quite sure what the use case is there; everything would be indented, with a single space before every entry.

> indent its children 1 space

That isn't possible. Once again, I'm not really seeing the use case, as everything but the root folder will be indented.

Pragmatic solution: use a sign column with width 1: set `view.signcolumn` to `1` or `auto:1`

**Reply by charbelnicolas** [reply]
> > indent the root folder 1 space
> 
> I'm not quite sure what the use case is there; everything would be indented, with a single space before every entry.
> 
> > indent its children 1 space
> 
> That isn't possible. Once again, I'm not really seeing the use case, as everything but the root folder will be indented.
> 
> Pragmatic solution: use a sign column with width 1: set `view.signcolumn` to `1` or `auto:1`

Children folders usually come indented below their parent folder which is not the case with NvimTree... That's why I wanted to add an indent. I think Signcolumn 1 or auto:1 adds 2 spaces instead of 1. In any case, customization is not always a bad thing (look at Neo tree)...

**Reply by alex-courtis** [reply]
> Children folders usually come indented below their parent folder which is not the case with NvimTree

Fair enough. Please raise a [feature request](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=feature+request&template=feature_request.md&title=)

> customization is not always a bad thing (look at Neo tree)...

Thank you for your feedback.

**Reply by charbelnicolas** [reply]
> > Children folders usually come indented below their parent folder which is not the case with NvimTree
> 
> Fair enough. Please raise a [feature request](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=feature+request&template=feature_request.md&title=)
> 
> > customization is not always a bad thing (look at Neo tree)...
> 
> Thank you for your feedback.

Thank you! I already did and you already replied!

https://github.com/nvim-tree/nvim-tree.lua/issues/2023
